patches/0005-Replace-getdents64-syscall-with-libc-call.patch

```diff
@@ -0,0 +1,26 @@
+From f607d18e9ee479dd30d4c1f737c08f2d7642122a Mon Sep 17 00:00:00 2001
+From: Andrei Tatar <andrei@unikraft.io>
+Date: Mon, 10 Jul 2023 16:23:27 +0200
+Subject: [PATCH 5/5] Replace getdents64 syscall with libc call
```

Comment:
Why is this needed?

Reply:
The patch itself? To be honest I no longer remember, it's been a long time. I did some basic tests with the two syscall replacing patches removed and it seems ok, so must've been a fix for an issue that's no longer relevant. I'll remove the patches but keep them on hand in case issues spring back up.